save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/OrganizationView.kt

```diff
@@ -239,18 +256,46 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
                                 }
                                 +"Description"
                             }
-                            button(classes = "btn btn-link text-xs text-muted text-left ml-auto") {
-                                +"Edit  "
-                                fontAwesomeIcon(icon = faEdit)
-                                attrs.onClickFunction = {
-
+                            if ((state.roleInOrganization?.priority ?: 0) >= 2) {
```

Comment:
very strange logic. You should rely on `isEditDisabled`. You have created a special state for it.